src/modules/listClass.js

```diff
@@ -0,0 +1,133 @@
+    this.list.forEach((task) => {
+      if (task.index === index) {
+        task.description = newDescription;
+      }
+    });
+    localStorage.setItem('localData', JSON.stringify(this.list));
+  }
+
+  removeTask(index) {
+    this.list = JSON.parse(localStorage.getItem('localData'));
+    this.list = this.list.filter((task) => task.index !== index);
+    localStorage.setItem('localData', JSON.stringify(this.list));
+    window.location = window.location.pathname;
+  }
+
+  render() {
+    this.list = JSON.parse(localStorage.getItem('localData'));
+    for (let i = 0; i < this.list.length; i += 1) {
+      const listElement = document.createElement('li');
+
+      listElement.classList.add('list-task');
+
+      const inputList = document.createElement('input');
+      inputList.type = 'checkbox';
+      inputList.id = 'task';
+      inputList.checked = this.list[i].completed;
+      listElement.appendChild(inputList);
+
+      const descriptionTask = document.createElement('label');
+      descriptionTask.classList.add('task-content');
+      descriptionTask.contentEditable = 'true';
+      descriptionTask.textContent = `${this.list[i].description}`;
+      listElement.appendChild(descriptionTask);
+
+      const buttonRemove = document.createElement('button');
+      buttonRemove.classList.add('button-more');
+      buttonRemove.style.display = 'none';
+      listElement.appendChild(buttonRemove);
+
+      const imageTrash = document.createElement('img');
+      imageTrash.src = trashIcon;
+      imageTrash.classList.add('image-button-more');
+      buttonRemove.appendChild(imageTrash);
+
+      const buttonTask = document.createElement('button');
+      buttonTask.classList.add('button-more');
+      listElement.appendChild(buttonTask);
+
+      const buttonImage = document.createElement('img');
+      buttonImage.src = Icon;
+      buttonImage.classList.add('image-button-more');
+      buttonTask.appendChild(buttonImage);
+
+      descriptionTask.addEventListener('focus', () => {
+        listElement.classList.add('editable-task');
+        buttonTask.style.display = 'none';
+        buttonRemove.style.display = 'inherit';
+      });
+
+      descriptionTask.addEventListener('blur', () => {
+        listElement.classList.remove('editable-task');
+        buttonRemove.style.display = 'none';
+        buttonTask.style.display = 'inherit';
+        if (descriptionTask.textContent === '') {
+          this.removeTask(this.list[i].index);
+        } else {
+          this.modifiedTask(descriptionTask.textContent, this.list[i].index);
+        }
+        descriptionTask.contentEditable = 'true';
+      });
+
+      descriptionTask.addEventListener('keypress', (e) => {
+        if (e.key === 'Enter') {
+          descriptionTask.contentEditable = 'false';
+        }
+      });
+
+      buttonRemove.addEventListener('click', () => {
+        if (descriptionTask.textContent === '') {
+          this.removeTask(this.list[i].index);
+        }
```

Comment:
- [ ] Great work on creating the function to remove tasks from the list :+1:. However, it seems that there might be an issue with its functionality, as indicated in the video below. When the user attempts to remove an element, it does not appear to be getting removed from the list. This issue needs to be addressed to ensure the proper functioning of the feature.


https://user-images.githubusercontent.com/105117832/236053904-6345a10d-44bb-4fb8-9cdb-db2e58b37a0e.mp4